Application/Education/Certificate/Setting/Frontend.php

```diff
@@ -399,6 +479,60 @@ public function frontendApproval($Data = null)
         return $Stage;
     }
 
+    public function frontendImplement()
+    {
+
+        $Stage = new Stage('Einstellungen', 'Zeugnisvorlagen installieren');
+        $Stage = self::setSettingMenue($Stage, '');
+
+        $Stage->setContent(
+            new Layout(
+                new LayoutGroup(array(
+                    new LayoutRow(array(
+                        new LayoutColumn(new Title('Auswahl'))
+                    )),
+                    new LayoutRow(array(
+                        new LayoutColumn(
+                            $this->getCertificateInstallButton(TblCertificate::CERTIFICATE_TYPE_PRIMARY, 'Zeugnisse Grundschule', 'GsJa')
+                            .$this->getCertificateInstallButton(TblCertificate::CERTIFICATE_TYPE_SECONDARY, 'Zeugnisse Oberschule', 'MsJ')
+                            .$this->getCertificateInstallButton(TblCertificate::CERTIFICATE_TYPE_GYM, 'Zeugnisse Gymnasium', 'GymJ')
+                            .$this->getCertificateInstallButton(TblCertificate::CERTIFICATE_TYPE_BERUFSFACHSCHULE, 'Zeugnisse Berufsfachschule', 'BfsHj')
+                        )
+                    ))
+                ))
+            )
+        );
+
+        return $Stage;
+    }
+
+    /**
+     * @param string $Type
+     * @param string $Name
+     * @param string $CertificateClass
+     *
+     * @return Standard|SuccessLink
+     */
+    private function getCertificateInstallButton($Type, $Name, $CertificateClass)
+    {
+
+        $Button = new Standard($Name, '/Education/Certificate/Setting/ImplementCertificate',
+            new Save(), array('Type' => $Type));
+        // Installation schon vorhanden? Test an einem Zeugnis
+        if(Generator::useService()->getCertificateByCertificateClassName($CertificateClass)){
+            $Button = new SuccessLink($Name,'/Education/Certificate/Setting/ImplementCertificate',
+                new Ok(), array('Type' => $Type),
+                'Erneut Installieren (eventuell fehlende/neue ergänzen)');
+        }
+        return $Button;
+    }
+
+    public function frontendImplementCertificate($Type = '')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Incorrect spacing between argument "$Type" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5646354)